source/system-administrators/index.rst

```diff
@@ -180,6 +180,7 @@ Crafter Studio
    studio/create-site-with-link-to-remote-repo.rst
    studio/session-timeout-settings.rst
    studio/publishing-blacklist.rst
+   upgrade/upgrading-in-context-editing.rst
```

Comment:
@alhambrav @sumerjabri please include in the right index. I used vita's suggestion of adding under sys-admin/upgrade but maybe it fits better elsewhere — particularly in/with the new IA.

Reply:
Who's this action item for? If for me, I need more details to be able to execute.

Reply:
I put an @alhambrav in the comment